asciidoc.py

```diff
@@ -2234,6 +2234,41 @@ class Section:
     """Static methods and attributes only."""
     endtags = []  # Stack of currently open section (level,endtag) tuples.
     ids = []      # List of already used ids.
+    NON_NFKD_MAP = {u'\u0181': u'B', u'\u1d81': u'd', u'\u1d85': u'l', u'\u1d89': u'r', u'\u028b': u'v',
```

Comment:
I think it's fine to promote this to a module global, even if it's only used within `Section`. This will make it a bit easier to catch this for refactoring in the future once I start making this a proper python module instead of one giant file.